lib/topological_inventory/sync/worker.rb

```diff
@@ -20,7 +20,14 @@ def run
         initial_sync
 
         self.messaging_client = ManageIQ::Messaging::Client.open(messaging_client_opts)
-        messaging_client.subscribe_topic(subscribe_opts) { |message| perform(message) }
+        messaging_client.subscribe_topic(subscribe_opts) do |message|
+          begin
+            perform(message)
+          rescue => err
+            logger.error(err)
```

Comment:
I wonder if we should change the way we log this? I think using `logger.error(` creates 2 separate records in kibana, which is then a bit confusing, just having separate backtrace

We have this in many places though, so it would require bigger effort.

Reply:
Yeah that's a good point we should put the exception message and the backtrace in one record

Reply:
Maybe something we can put in manageiq-loggers similar to log_backtrace but also log the exception message